src/Mod/Surface/App/AppSurface.cpp

```diff
@@ -35,49 +40,70 @@
 #include <Base/Parameter.h>
 
 
-namespace Surface {
-class Module : public Py::ExtensionModule<Module>
+
+
+namespace Surface
+{
+class Module: public Py::ExtensionModule<Module>
 {
 public:
     Module() : Py::ExtensionModule<Module>("Surface")
     {
-        initialize("This module is the Surface module."); // register with Python
+        initialize("This module is the Surface module.");// register with Python
     }
 
     virtual ~Module() {}
 
 private:
 };
 
-PyObject* initModule()
+PyObject *initModule()
 {
     return Base::Interpreter().addModule(new Module);
 }
 
-} // namespace Surface
+}// namespace Surface
 
 
 /* Python entry */
 PyMOD_INIT_FUNC(Surface)
 {
+
+
     try {
-        Base::Interpreter().runString("import Part");
+      Base::Interpreter().runString("import Part");
     }
-    catch(const Base::Exception& e) {
+    catch (const Base::Exception &e) {
         PyErr_SetString(PyExc_ImportError, e.what());
         PyMOD_Return(nullptr);
     }
 
-    PyObject* mod = Surface::initModule();
+    PyObject *mod = Surface::initModule();
     Base::Console().Log("Loading Surface module... done\n");
+    //PyObject *surfaceModule = Surface::initModule();

```

Comment:
I don't think we should keep the commented code.

Reply:
Fixed.
Thank you.